manifests/check.pp

```diff
@@ -64,6 +67,7 @@
   $timeout    = undef,
   $token      = undef,
   $ttl        = undef,
+  $grpc       = undef,
```

Comment:
can you please add a suitable datatype for the new parameter?

Reply:
You mean like this?
```
  $token      = undef,
  $ttl            = undef,
  String[1]     $grpc       = undef,
```

Reply:
Something like this? 

```
  $token = undef,
  $ttl = undef,
  String[1] $grpc = undef,
```

Reply:
yes :)